test/integration/arch/inspec.yml

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+# vim: ft=yaml
+---
+name: default
+title: pycharm formula
+maintainer: SaltStack Formulas
+license: Apache-2.0
+summary: Verify that the pycharm formula is setup and configured correctly
```

Comment:
The name and summary should be appropriate for each suite, will be resolved when commit is added here:

https://github.com/myii/jetbrains-pycharm-formula/commit/a3bac46edc0678d4436b425e3929d5c04aea72f6#diff-e12e128804f69778e0d42bd5d468d7a8L4-R9